types/foundry/build/generate-client-base-mixes.ts

```diff
@@ -470,6 +470,12 @@ const clientDocs: Record<string, { hasSheet?: boolean; isCanvasDoc?: boolean; pa
     Wall: { isCanvasDoc: true },
 };
 
+// imports
+console.log(
+    String.raw`import type { ApplicationV2, DocumentSheetV2 } from "../../../client-esm/applications/api/module.d.ts";
+
+`,
+);
```

Comment:
I have no idead how this works so it's probably not an error, just checking that this is intended.

Reply:
I've looked at the rest of the file and this is intended after all.